src/App.js

```diff
@@ -1,12 +1,44 @@
-import React from 'react';
+import React, {useEffect, useState} from 'react';
 import './App.css';
+import axios from "axios";
+import PokemonCard from "./assets/componenten/PokemonCard";
 
 function App() {
-  return (
-    <div>
-      Begin hier met de opdracht!
-    </div>
-  );
+    const [pokemon, setPokemon] = useState([]);
```

Comment:
Zodra de setter aangeroepen wordt dan is het geen array mee. Ik zou dan de default waarde `undefined` maken